d_30/start.sh

```
@@ -1,15 +1,14 @@
-#!/bin/bash
+# sh start.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)